src/xrGame/DiscordRPC.cpp

```diff
@@ -0,0 +1,53 @@
+#include "stdafx.h"
+#include "DiscordRPC.h"
+#include <DiscordRPC/discord_register.h>
+#include <DiscordRPC/discord_rpc.h>
+#include <xrCore/Text/StringConversion.hpp>
```

Comment:
Using `<>` on our project includes is wrong, we should use quotes.
We also include third-party files after our own.
```suggestion
#include "DiscordRPC.h"
#include "xrCore/Text/StringConversion.hpp"

#include <DiscordRPC/discord_register.h>
#include <DiscordRPC/discord_rpc.h>
```